python/hail/keytable.py

```diff
@@ -690,6 +690,25 @@ def export_cassandra(self, address, keyspace, table, block_size=100, rate=1000):
         self._jkt.exportCassandra(address, keyspace, table, block_size, rate)
 
     @handle_py4j
+    @typecheck_method(host=strlike,
+                      port=integral,
+                      index=strlike,
+                      index_type=strlike,
+                      block_size=integral,
+                      config=nullable(dictof(strlike, strlike)),
+                      verbose=bool)
+    def export_elasticsearch(self, host, port, index, index_type, block_size, config=None, verbose=True):
+        """Export to Elasticsearch.
+
+        .. warning::
+
+          :py:meth:`~.export_elasticsearch` is EXPERIMENTAL.
```

Comment:
I think experimental.rst looks good here.  (Didn't know about that!  I like it.)

Not intentional on my part, but given that this is still getting fleshed out, I wasn't going to force it.